components/Topbar/index.tsx

```diff
@@ -35,15 +35,22 @@ const TopBar = () => {
         sx={{
           display: 'flex',
           justifyContent: 'center',
-          padding: '0 16px',
+          padding: '16px',
           width: '100%',
         }}
       >
         <Typography variant="modalRow" color="white">
-          {t('OP Mainnet upgrade to Bedrock release: June 6, 2023, 16:00 UTC. 2-4 hours downtime expected.')}{' '}
+          <Trans>
+            {t(
+              'The Optimism Mainnet upgrade to the Bedrock release will take place on <strong>{{date}}</strong> There will be 2-4 hours of downtime.',
+              {
+                date: t('June 6, 2023 at 16:00 UTC.'),
+              },
+            )}{' '}
```

Comment:
```suggestion
            )}
```
?

Reply:
This is to make space for the following sentence without forcing it into the translations.